blocks/footer-copyright/footer-copyright.css

```diff
@@ -1,9 +1,9 @@
 .footer-copyright {
   background-color: var(--color-background);
-  padding: 1.5rem var(--spacing-page-gutters) 3rem;
+  padding: 1.5rem var(--page-gutters) 3rem;
```

Comment:
I renamed these to page-level variables instead of spacing-specific ones, they're more layout anyway.